watchtower-plugin/src/constants.rs

```diff
@@ -3,6 +3,12 @@ pub const TOWERS_DATA_DIR: &str = "TOWERS_DATA_DIR";
 pub const DEFAULT_TOWERS_DATA_DIR: &str = ".watchtower";
 
 /// Collections of plugin option names, default values and descriptions
+pub const DEFAULT_SUBSCRIPTION_START: Option<i64> = None;
+pub const SUBSCRIPTION_START: &str = "subscription-start";
+pub const SUBSCRIPTION_START_DESC: &str = "subscription-start time";
+pub const DEFAULT_SUBSCRIPTION_EXPIRY: Option<i64> = None;
+pub const SUBSCRIPTION_EXPIRY: &str = "subscription-expiry";
+pub const SUBSCRIPTION_EXPIRY_DESC: &str = "subscription-expiry time";
```

Comment:
None of this is required. There is no such thing as a default subscription start or expiry. They depend on the block height at which the user requested the subscription.